client/src/components/pages/FilterPage/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+
+import axios from 'axios';
+import { notification } from 'antd';
+
+import Cards from '../../common/Card';
+import Filter from '../../common/Filter';
+import './style.css';
+
+class FilterPage extends Component {
+  state = {
+    data: [],
+  };
+
+  componentDidMount = async () => {
+    const result = await axios.get('/api/v1/intial');
+    this.setState(prevState => ({ ...prevState, data: result.data.data }));
+  };
+
+  openNotificationWithIcon = error => {
+    notification.error({
+      message: 'somthing wrong !!',
+      description: error.message,
+      duration: 3,
+    });
+  };
+
+  handleSubmit = async data => {
+    try {
+      const result = await axios.post('/api/v1/filter', { data: { ...data } });
+      this.setState(prevState => ({ ...prevState, data: result.data.data }));
+    } catch (error) {
+      this.openNotificationWithIcon(error);
+    }
+  };
+
+  render() {
+    const { data } = this.state;
+    if (!data.length) return <h3>Loding...</h3>;
```

Comment:
```suggestion
    if (!data.length) return <Loader />;
```
add Loader component instead of this html heading